### ISSUE TITLE
Track Time does not Show anything above one 59:59

### ISSUE THREAD

**Issue by superpunkduck** [pathofleastresistor/polr-ytube-media-card#9]
I have a track playing that is 01:02:56 
1 Hour 2 Minutes 56 Seconds Long... but the track counter on left and right only shows minutes:seconds
![image](https://github.com/pathofleastresistor/polr-ytube-media-card/assets/50434137/a90276d2-ca51-4de9-b4ae-d730466af0b0)

**Comment by pathofleastresistor** [comment]
Yes - that is the current limitation. I have to update my method of converting seconds to MM:SS.
